platform/c2000/inverter_drivers/pmicdriver.h

```diff
@@ -100,36 +92,16 @@ static const uint16_t WatchdogResponsesSize =
 
 static const int StateTransitionDelay = 100; // uS
 
-
 /**
  * \brief Verify that function x didn't fail. Return immediately if it did.
  * Assumes a local variable "result" of type TeslaM3PowerWatchdog::Error
  */
 #define CHECK(x)                                                               \
```

Comment:
With the move to a header only this macro is now leaking to any users of the class which could cause odd results. It does make the code more readable I think. Perhaps #undef at the end of the header?